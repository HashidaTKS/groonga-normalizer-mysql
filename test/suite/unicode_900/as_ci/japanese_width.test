register normalizers/mysql

normalize \
  'NormalizerMySQLUnicode900("weight_level", 2)' \
  "ï¾Š"
