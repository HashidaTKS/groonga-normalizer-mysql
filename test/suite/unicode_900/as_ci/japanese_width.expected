register normalizers/mysql
[[0,0.0,0.0],true]
normalize   'NormalizerMySQLUnicode900("weight_level", 2)'   "ﾊ"
[[0,0.0,0.0],{"normalized":"は","types":[],"checks":[]}]
